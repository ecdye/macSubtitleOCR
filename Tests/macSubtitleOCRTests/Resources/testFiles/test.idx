# VobSub index file, v7 (do not modify this line!)
palette: 000000, 0000ff, 00ff00, ff0000, ffff00, ff00ff, 00ffff, ffffff, 808000, 8080ff, 800080, 80ff80, 008080, ff8080, 555555, aaaaaa
langidx: 0

id: en, index: 0
timestamp: 00:01:47:250, filepos: 000000000
timestamp: 00:01:51:792, filepos: 000001800
timestamp: 00:01:58:000, filepos: 000003000
timestamp: 00:02:01:750, filepos: 000006000
timestamp: 00:02:05:250, filepos: 000008000
timestamp: 00:02:07:500, filepos: 000009000
timestamp: 00:02:09:417, filepos: 000009800
timestamp: 00:02:15:000, filepos: 00000c000
timestamp: 00:02:18:000, filepos: 00000d800
timestamp: 00:02:23:417, filepos: 00000f800
timestamp: 00:02:28:833, filepos: 000010800
timestamp: 00:02:32:958, filepos: 000012000
timestamp: 00:03:27:250, filepos: 000014000
timestamp: 00:03:30:750, filepos: 000015800
timestamp: 00:03:48:250, filepos: 000016800
timestamp: 00:04:10:333, filepos: 000017800
timestamp: 00:04:25:250, filepos: 000019000
timestamp: 00:05:04:000, filepos: 000019800
timestamp: 00:05:38:750, filepos: 00001a800
timestamp: 00:07:25:833, filepos: 00001b000
timestamp: 00:07:32:792, filepos: 00001c000
timestamp: 00:07:37:792, filepos: 00001d800
timestamp: 00:07:40:833, filepos: 00001f000
timestamp: 00:09:17:583, filepos: 000020800
timestamp: 00:10:21:583, filepos: 000021000
timestamp: 00:10:26:208, filepos: 000021800
