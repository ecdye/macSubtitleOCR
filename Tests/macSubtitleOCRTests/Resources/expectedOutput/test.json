[
  {
    "image" : 1,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 52,
        "text" : "This blade has a dark past.",
        "width" : 680,
        "x" : 619,
        "y" : 0
      }
    ],
    "text" : "This blade has a dark past."
  },
  {
    "image" : 2,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 41,
        "text" : "It has shed much innocent blood.",
        "width" : 828,
        "x" : 544,
        "y" : 0
      }
    ],
    "text" : "It has shed much innocent blood."
  },
  {
    "image" : 3,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 59,
        "text" : "You're a fool for traveling alone,",
        "width" : 789,
        "x" : 566,
        "y" : 0
      },
      {
        "confidence" : 1,
        "height" : 53,
        "text" : "so completely unprepared.",
        "width" : 672,
        "x" : 619,
        "y" : 64
      }
    ],
    "text" : "You're a fool for traveling alone,\nso completely unprepared."
  },
  {
    "image" : 4,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 52,
        "text" : "You're lucky your blood's still flowing.",
        "width" : 918,
        "x" : 502,
        "y" : 0
      }
    ],
    "text" : "You're lucky your blood's still flowing."
  },
  {
    "image" : 5,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 52,
        "text" : "Thank you.",
        "width" : 281,
        "x" : 817,
        "y" : 0
      }
    ],
    "text" : "Thank you."
  },
  {
    "image" : 6,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 43,
        "text" : "So...",
        "width" : 114,
        "x" : 901,
        "y" : 0
      }
    ],
    "text" : "So..."
  },
  {
    "image" : 7,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 59,
        "text" : "What brings you to",
        "width" : 474,
        "x" : 722,
        "y" : 0
      },
      {
        "confidence" : 1,
        "height" : 53,
        "text" : "the land of the gatekeepers?",
        "width" : 717,
        "x" : 600,
        "y" : 64
      }
    ],
    "text" : "What brings you to\nthe land of the gatekeepers?"
  },
  {
    "image" : 8,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 52,
        "text" : "I'm searching for someone.",
        "width" : 680,
        "x" : 619,
        "y" : 0
      }
    ],
    "text" : "I'm searching for someone."
  },
  {
    "image" : 9,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 54,
        "text" : "Someone very dear?",
        "width" : 519,
        "x" : 700,
        "y" : 0
      },
      {
        "confidence" : 1,
        "height" : 54,
        "text" : "A kindred spirit?",
        "width" : 410,
        "x" : 753,
        "y" : 63
      }
    ],
    "text" : "Someone very dear?\nA kindred spirit?"
  },
  {
    "image" : 10,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 60,
        "text" : "A dragon.",
        "width" : 249,
        "x" : 835,
        "y" : -1
      }
    ],
    "text" : "A dragon."
  },
  {
    "image" : 11,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 52,
        "text" : "A dangerous quest for a lone hunter.",
        "width" : 912,
        "x" : 505,
        "y" : 0
      }
    ],
    "text" : "A dangerous quest for a lone hunter."
  },
  {
    "image" : 12,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 47,
        "text" : "I've been alone for",
        "width" : 468,
        "x" : 725,
        "y" : 0
      },
      {
        "confidence" : 1,
        "height" : 53,
        "text" : "as long as I can remember.",
        "width" : 683,
        "x" : 616,
        "y" : 64
      }
    ],
    "text" : "I've been alone for\nas long as I can remember."
  },
  {
    "image" : 13,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 43,
        "text" : "We're almost done. Shhh...",
        "width" : 683,
        "x" : 616,
        "y" : 0
      }
    ],
    "text" : "We're almost done. Shhh..."
  },
  {
    "image" : 14,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 56,
        "text" : "Hey, sit still.",
        "width" : 299,
        "x" : 811,
        "y" : -2
      }
    ],
    "text" : "Hey, sit still."
  },
  {
    "image" : 15,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 53,
        "text" : "Good night, Scales.",
        "width" : 488,
        "x" : 717,
        "y" : 0
      }
    ],
    "text" : "Good night, Scales."
  },
  {
    "image" : 16,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 50,
        "text" : "Get him, Scales! Come on!",
        "width" : 680,
        "x" : 619,
        "y" : 0
      }
    ],
    "text" : "Get him, Scales! Come on!"
  },
  {
    "image" : 17,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 43,
        "text" : "Scales?",
        "width" : 203,
        "x" : 856,
        "y" : 0
      }
    ],
    "text" : "Scales?"
  },
  {
    "image" : 18,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 43,
        "text" : "Yeah! Come on!",
        "width" : 410,
        "x" : 756,
        "y" : 0
      }
    ],
    "text" : "Yeah! Come on!"
  },
  {
    "image" : 19,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 43,
        "text" : "Scales!",
        "width" : 186,
        "x" : 867,
        "y" : 0
      }
    ],
    "text" : "Scales!"
  },
  {
    "image" : 20,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 42,
        "text" : "I have failed.",
        "width" : 326,
        "x" : 795,
        "y" : 0
      }
    ],
    "text" : "I have failed."
  },
  {
    "image" : 21,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 52,
        "text" : "You've only failed to see...",
        "width" : 653,
        "x" : 633,
        "y" : 0
      }
    ],
    "text" : "You've only failed to see..."
  },
  {
    "image" : 22,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 53,
        "text" : "These are dragon lands, Sintel.",
        "width" : 781,
        "x" : 569,
        "y" : 0
      }
    ],
    "text" : "These are dragon lands, Sintel."
  },
  {
    "image" : 23,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 52,
        "text" : "You are closer than you know.",
        "width" : 753,
        "x" : 583,
        "y" : 0
      }
    ],
    "text" : "You are closer than you know."
  },
  {
    "image" : 24,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 43,
        "text" : "Scales!",
        "width" : 186,
        "x" : 867,
        "y" : 0
      }
    ],
    "text" : "Scales!"
  },
  {
    "image" : 25,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 43,
        "text" : "Scales?",
        "width" : 203,
        "x" : 856,
        "y" : 0
      }
    ],
    "text" : "Scales?"
  },
  {
    "image" : 26,
    "lines" : [
      {
        "confidence" : 1,
        "height" : 45,
        "text" : "Scales...",
        "width" : 212,
        "x" : 853,
        "y" : -1
      }
    ],
    "text" : "Scales..."
  }
]